### PR TITLE
Code from Sprint 1 & 2

### PR DESCRIPTION
**Pull request by Bsharma4**
Guys this is the code from Sprint 1 and 2 that we worked on!

### CODE REVIEW

File: Webpage.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html/>
```
Comment (MarshallOkafor) [inline Webpage.html:1]: I think this file has already being modified and uploaded. I do not think we should add this file again.

File: index.css [hunk 2]
```diff
@@ -1,101 +1,101 @@
-/* This is to use a unified border size */
```
Comment (MarshallOkafor) [inline index.css:1]: I do not think that this file needs to be removed or modified unless there is a hotfix.

File: index.css [hunk 3]
```diff
@@ -1,101 +1,101 @@
...
-
-.suggest-text {
- getTextBox();
-}
-
-function getTextBox(){
-    String k = document.getElementById('Etext').value;
-    return k;
-  }
+/* This is to use a unified border size */
```
Comment (MarshallOkafor) [inline index.css:1]: I do not think that this file needs to be removed or modified unless there is a hotfix.

File: Spellcheckmain-errordetection.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            incorrectwords.append(word)
+    return incorrectwords
+
+#Printing errors - Temporary function
+def printerrors(errorlist):
+    print("The misspelled words are: ")
+    for word in errorlist:
+        print(word)
+
+
```
Comment (MarshallOkafor) [inline Spellcheckmain-errordetection.py:38]: Great! This appears to mean the main function initiating the process to take the words and return the error list of words.

File: Spellcheckmain-errordetection.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
+#SpellChecker
```
Comment (MarshallOkafor) [inline Spellcheckmain-errordetection.py:1]: Great! This entire program handles the processing of the words and returning the wrong list of words.

File: Dictionary-Spellcheckmain-errordetection.txt [hunk 6]
```diff
@@ -0,0 +1,18 @@
+This
```
Comment (MarshallOkafor) [inline Dictionary-Spellcheckmain-errordetection.txt:1]: This looks like a temporary dictionary file you used. Great!

File: index.html [hunk 7]
```diff
@@ -1,56 +1,55 @@
-<!DOCTYPE html/>
```
Comment (MarshallOkafor) [inline index.html:1]: I do not think that this file needs to be removed or modified unless there is a hotfix.

**Review by MarshallOkafor — COMMENTED** [review]
Great and thanks Harry for the pull requests. However, I made some reviews that I think you should check and correct.

1. Resend the Dictionary and sample txt files for merging.
2. Resend the python error word detection file for merging.
3. The HTML and CSS files do not need any modification again for now except there is a bug fix, hotfix or additional feature.

Once you are done with addressing the reviews I have made, resend a pull request so that we can review again and merge.

Thank you.

File: Webpage.html (hunk 1 above)
Comment (SUDANGOKUL) [reply Webpage.html:1]: Yeah, We are  done with this already so no need to do this again.

File: index.css (hunk 2 above)
Comment (SUDANGOKUL) [reply index.css:1]: it's a comment though, what differences does it make let's leave it back.

File: index.css (hunk 3 above)
Comment (SUDANGOKUL) [reply index.css:1]: I'm with you on this @MarshallOkafor

File: Spellcheckmain-errordetection.py (hunk 4 above)
Comment (SUDANGOKUL) [reply Spellcheckmain-errordetection.py:38]: 👍

**Review by SUDANGOKUL — APPROVED** [review]
Well I have gone through them and they are all set and ready to go

**Review by SUDANGOKUL — CHANGES_REQUESTED** [review]
Bro roll It back if we merge it , it will take us back again.

**Pull request closed without merging**